functions.js

```diff
@@ -0,0 +1,343 @@
+// ## Question 1
```

Comment:
Good job with the lab. Only minor feedback I have is 4 space indentation which is more for language like Python and such. Does your IDE do that automatically?